docs/UserGuide.md

```diff
@@ -956,6 +956,201 @@ Important notes:
 > > <img src="images/unicash/command-outputs/addTransactionFailedOutput3.png" width="1000" />
 
 #### 4.1.2 EditTransactionCommand
+Edits an existing transaction in UniCa$h.
+
+Command: `edit_transaction INDEX [n/NAME] [type/TYPE] [amt/AMOUNT] [dt/DATETIME] [l/LOCATION] [c/CATEGORY]`
+
+Command Options:
+
+| Option Name | Optional? | Purpose                                                                                       |
+|-------------|-----------|-----------------------------------------------------------------------------------------------|
+| INDEX       | No        | (One-based) index of the transaction to edit.                                                 |
+| n/          | Yes       | Name of the transaction.                                                                      |
+| type/       | Yes       | Transaction type of transaction.<br/>Valid types are `income` and `expense`.                  |
+| amt/        | Yes       | Monetary amount of transaction. Has to be a positive value.                                   |
+| dt/         | Yes       | Date and time where transaction was made.<br/>Defaults to current date time if not specified. |
+| l/          | Yes       | Location where transaction was made.<br/>Defaults to `''` if not specified.                   |
+| c/          | Yes       | Category tagged to that transaction.<br/>No categories tagged if not specified.               |
+
+Important notes:
+1. There is a character limit for `Name` and `Location` set at up to 500 characters.
+2. `Amount` entered has to be positive for both `income` and `expense`.
+3. `Amount` is automatically rounded to 2 decimal places.
+4. `Datetime` entered can be in the `dd-MM-uuuu HH:mm` (e.g. 31-10-2023 12:00), `uuuu-MM-dd HH:mm` (e.g. 2023-10-31 22:59), or `dd MMM uuuu HH:mm` (e.g. 31 Oct 2023 22:59) formats.
```

Comment:
Maybe we can add this in an appendix, because this datetime format applies to more than just the `editTransaction` command.

Reply:
Agreed! 😄  I was thinking of creating a section to list out the constraints for each field, since after the discussion today, it seems that there have been more constraints implemented. However, should we leave it as it is for the PE dry run tomorrow, and work on it in the coming week for v1.4?